test/feeds.test.js

```diff
@@ -132,3 +132,43 @@ describe('test POST /feeds endpoint', () => {
     expect(res.status).toEqual(409);
   });
 });
+
+describe.skip('test DELETE /feeds/:id endpoint', () => {
+  let feedId;
+
+  beforeEach(async () => {
+    logout();
+    const user = login('Johannes Kepler', 'user1@example.com');
+    const feedData = {
+      author: user.name,
+      url: 'http://telescope200.cdot.systems',
+      user: user.id,
+    };
+    const res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
+    expect(res.status).toEqual(201);
+    feedId = res.id;
+  });
+
+  afterAll(() => logout());
+
+  it('should respond with a 403 status when not logged in', async () => {
+    logout();
+    const res = await request(app).delete(`/feeds/${feedId}`);
+    expect(res.status).toEqual(403);
+  });
+
+  it('should respond with a 403 status when targeted feed is not owned by user', async () => {
+    logout();
+    login('Galileo Galilei', 'user2@example.com');
+    const res = await request(app).delete(`/feeds/${feedId}`);
+    expect(res.status).toEqual(403);
+  });
+
+  it('should respond with a 204 status when targeted feed is owned by user', async () => {
+    let res = await request(app).delete(`/feeds/${feedId}`);
```

Comment:
Let's explicitly do the login again as the correct user, so the test is clear that it's expecting that user to be the current user (your test is right now, but it would be more future-proof to add it here):

```js
const user = login('Johannes Kepler', 'user1@example.com');
```

Reply:
Fixed in https://github.com/Seneca-CDOT/telescope/pull/958/commits/5cee6b4c37a89f88fa372ea01b50ff1e5cf45b72